spec/unit/middleware/requeue_exceptions_spec.rb

```diff
@@ -16,8 +16,8 @@ def around_perform(job)
         end
       end
 
-      let(:container) do 
-        container = container_class.new 
+      let(:container) do
+        container = container_class.new
```

Comment:
It is a side effect. I updated file, and the editor did cleanup.

There is three setting to editor (not all editors support all of them): (a) ignore tailing whitespaces, (b) kill them, and (c) kill only in modified lines. There are also different team policies. I understand your frustration, and would propose to a global whitespace cleanup with lint check afterwards. Should we take it to retrospective?